jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/impl/CommonUtilsTest.java

```diff
@@ -50,6 +56,68 @@
             new TypeAnnotationsScanner(), 
             new FieldAnnotationsScanner(), new SubTypesScanner());
 
+    private List<ProcessDefinition> expected;
+    private List<ProcessDefinition> input;
+
+    @Before
+    public void initComparatorTest(){
+        expected = new ArrayList<ProcessDefinition>() {{
```

Comment:
@bxf12315 I would recommend using `Arrays.asList(...)`  instead of creating the list using inner add.

Reply:
ok